.env.test

```diff
@@ -1,3 +1,4 @@
+ALLOW_BLOCK_MINED_POINTS=true
```

Comment:
Can this be added to [`.env.template`](https://github.com/iron-fish/ironfish-api/blob/master/.env.template) too?

Reply:
yup, done